src/components/section/BaseModal/BaseModal.tsx

```diff
@@ -0,0 +1,39 @@
+import { createPortal } from 'react-dom';
+import { MouseEvent, ReactNode } from 'react';
+import styles from './BaseModal.module.css';
+import Image from 'next/image';
+
+interface Props {
+  closeModal: () => void;
+  children: ReactNode;
+}
+
+export default function BaseModal({ closeModal, children }: Props) {

```

Comment:
컴포넌트 props에 children이 있는 경우는 더 간단히 PropsWithChildren 유틸타입을 사용하셔도 됩니다.
```suggestion
interface Props {
  closeModal: () => void;
}

export default function BaseModal({ closeModal, children }: PropsWithChildren<Props>) {
```